xmtp_mls/src/identity_updates.rs

```diff
@@ -259,9 +267,14 @@ where
         installation_ids: Vec<Vec<u8>>,
     ) -> Result<SignatureRequest, ClientError> {
         let inbox_id = self.inbox_id();
-        let current_state = self
-            .get_association_state(&self.store().conn()?, &inbox_id, None)
-            .await?;
+
+        let current_state = retry_async!(
+            Retry::default(),
+            (async {
+                self.get_association_state(&self.store().conn()?, &inbox_id, None)
+                    .await
+            })
+        )?;
```

Comment:
This should cover the case converse is hitting when the database is trying to reconnect after getting a signature from a different app for revoking installations.